Lib/_pyrepl/unix_console.py

```diff
@@ -39,6 +40,13 @@
 from .unix_eventqueue import EventQueue
 from .utils import wlen
 
+# declare posix optional to allow None assignment on other platforms
```

Comment:
I think it's just easier to have a `# type: ignore[...]` instead of a type annotation.

Reply:
for what it's worth, doing so requires two mypy ignores, one at import time and one for the `posix is not None` check, which triggers `[redundant]`

Reply:
Ah, I thought it was a single ignore that would be needed =/ I will leave it to @pablogsal's choice as he's one the REPL maintainers. I would honestly prefer keeping two `# type: ignore[]` because that saves an import to `types` (*if* we keep the type annotation, I think *every* REPL session will import `types` (if the latter is already imported, it doesn't matter but I don't think it's imported))

Reply:
iirc, `types` is either a) not a real module or b) already loaded at interpreter start, so this shouldn't be a big deal. a rudimentary test:

```
>>> timeit.timeit("import sys")
0.3161898439284414
>>> timeit.timeit("import types")
0.05900560016743839
```

Reply:
it does not show `types`, but `python -X importtime -i -c 'pass'` does 

```
import time:      1098 |       1098 | readline
import time:        32 |         32 |   atexit
import time:       251 |        251 |           types
```

so it's probably fine in this context?

Reply:
Oh ok it's fine. It's imported because of `ast` which is needed by the REPL itself I think, right? So we can leave the import to `types` where it's needed.

Reply:
The unix_console changes look fine to me.